packages/backend-core/src/security/encryption.ts

```diff
@@ -8,7 +8,7 @@ const RANDOM_BYTES = 16
 const STRETCH_LENGTH = 32
 
 export enum SecretOption {
-  JWT = "jwt",
+  API = "api",
```

Comment:
How will this work in self host? Given the secret doesn't exist there - we probably need to consider using the `JWT_SECRET` as a fallback right? We should probably also update this for new installs.

Reply:
That's what is does atm, always falls back to the JWT secret value. I'll add it for new installs too just incase someone does ever want to change their JWT secret